code/src/java/pcgen/cdom/base/CDOMObject.java

```diff
@@ -230,7 +230,7 @@ public final Formula remove(FormulaKey key)
 
 	public final Set<FormulaKey> getFormulaKeys()
 	{
-		return formulaChar == null ? Collections.<FormulaKey> emptySet()
+		return formulaChar == null ? Collections.emptySet()
```

Comment:
I retract my concern.  Java 8 is smarter than previous versions and can correctly derive the type without a warning.